controllers/change_coordinators.go

```diff
@@ -46,13 +46,13 @@ func (c changeCoordinators) reconcile(ctx context.Context, r *FoundationDBCluste
 
 	adminClient, err := r.getDatabaseClientProvider().GetAdminClient(cluster, r)
 	if err != nil {
-		return &requeue{curError: err}
+		return &requeue{curError: err, delayedRequeue: true}
 	}
 	defer adminClient.Close()
 
 	status, err := adminClient.GetStatus()
 	if err != nil {
-		return &requeue{curError: err}
+		return &requeue{curError: err, delayedRequeue: true}
```

Comment:
I don't follow why "delayedRequeue" is getting added here, and its connection to version compatible upgrades.

Reply:
It's not really related to this change an I can revert it. In general we want to delay the requeue everywhere to make sure a single step is not blocking the operator from making progress and potentially dead lock itself.

Reply:
You can keep the change, that's fine.